packages/sit-onyx/src/components/OnyxFileUpload/examples/FileUploadModalExample.vue

```diff
@@ -0,0 +1,71 @@
+<script lang="ts" setup>
+import { iconTrash } from "@sit-onyx/icons";
+import { ref } from "vue";
+import OnyxFileCard from "../../OnyxFileCard/OnyxFileCard.vue";
+import OnyxIconButton from "../../OnyxIconButton/OnyxIconButton.vue";
+import OnyxModal from "../../OnyxModal/OnyxModal.vue";
+import OnyxSystemButton from "../../OnyxSystemButton/OnyxSystemButton.vue";
+import OnyxFileUpload from "../OnyxFileUpload.vue";
+
+const allFiles = ref<File[]>([]);
+const isOpen = ref(false);
+
+const removeFile = (fileToRemove: File) => {
+  allFiles.value = allFiles.value.filter((file) => file !== fileToRemove);
+};
+</script>
+
+<template>
+  <OnyxFileUpload v-model="allFiles" :multiple="true"> </OnyxFileUpload>
```

Comment:
```suggestion
  <OnyxFileUpload v-model="allFiles" :multiple="true" list-type="hidden" />
```

When we show a modal, the default fiel list should be hidden